ajax/libs/timelinejs/package.json

```diff
@@ -1,28 +1,15 @@
 {
   "name": "timelinejs",
   "filename": "js/timeline-min.js",
-  "version": "2.36.0",
+  "version": "3.4.7",
   "description": "TimelineJS is an open-source tool that enables you to build visually-rich interactive timelines and is available in 40 languages.",
   "homepage": "https://timeline.knightlab.com/",
   "keywords": [
     "timeline"
```

Comment:
Add some keywords to it. Ref: https://github.com/NUKnightLab/TimelineJS/blob/master/bower.json